contrib/kafka/filters/network/source/protocol/generator.py

```diff
@@ -534,7 +534,7 @@ class Primitive(TypeSpecification):
     # Custom values that make test code more readable.
     KAFKA_TYPE_TO_EXAMPLE_VALUE_FOR_TEST = {
         'string':
-            '"string"',
+            'static_cast<std::string>("string")',
```

Comment:
FWIW This is necessary for test code (example values) to compile (as with changing request declarations we ran into a situation where constructor invocations are ambiguous in https://github.com/envoyproxy/envoy/blob/v1.21.0/contrib/kafka/filters/network/test/protocol/response_utilities_cc.j2#L43).